protocol/0057-TRAN-transfers.md

```diff
@@ -264,11 +267,16 @@ message CancelTransfer {
   - If I have enough funds to pay transfer and fees, the transfer happens.
   - If I do not have enough funds to pay transfer and fees, the transfer is cancelled.
   - The fees are being paid into the infrastructure pool
+  - The transfer fee discount is correctly applied with network parameter`transfer.feeDiscountDecay`(<a name="0057-TRAN-014" href="#0057-TRAN-014">0057-TRAN-014</a>)
+  - The fee-free transfer amount is accessible while API (<a name="0057-TRAN-017" href="#0057-TRAN-017">0057-TRAN-017</a>)
 - As a user I can do a transfer from any of the valid accounts (I control them and they're a valid source), and fees are taken from the source account when the transfer is executed (when `transfer amount * transfer.fee.factor > transfer.fee.maxQuantumAmount * quantum`). (<a name="0057-TRAN-011" href="#0057-TRAN-011">0057-TRAN-011</a>)
-  - The fee cost is correctly calculated using the network parameter `transfer.fee.Maxfactor`
+  - The fee cost is correctly calculated using the network parameter
   - If I have enough funds to pay transfer and fees, the transfer happens.
   - If I do not have enough funds to pay transfer and fees, the transfer is cancelled.
   - The fees are being paid into the infrastructure pool
+  - The transfer fee discount is correctly applied with network parameter`transfer.feeDiscountDecay`(<a name="0057-TRAN-015" href="#0057-TRAN-015">0057-TRAN-015</a>)
+  - The fee-free transfer amount is accessible while API (<a name="0057-TRAN-018" href="#0057-TRAN-018">0057-TRAN-018</a>)
+  - When transfer fee discount amount is less than `M x quantum` (M is network parameter `transfer.feeDiscountMinimumTrackedAmount`), then no transfer fee discount will be applied (<a name="0057-TRAN-016" href="#0057-TRAN-016">0057-TRAN-016</a>)
```

Comment:
The `transfer.feeDiscountMinimumTrackedAmount` parameter is for setting the stored fee discount to 0 I believe, so I think the AC for it should be something like "fee discount saved up before epoch end was X > M * quantum, then epoch end happened and discount factor was applied, now X * D < M * quantum so the discount balance is set to 0"